### ISSUE TITLE
[BUG] MTU is overwrite by storage network

### ISSUE THREAD

**Issue by guangbochen** [harvester/harvester#4297]
**Describe the bug**

**To Reproduce**
1. Go to the Harvester network page and add a new clusterNetwork and networkconfig with MUT 9000
    <img width="2183" alt="image" src="https://github.com/harvester/harvester/assets/4569037/bf578128-b0fa-43ee-b1bd-b5805a5cd5f6">
1. the current MTU of both the bridge and bond interface is `9000`
1. config storage network on the settings page and select the above configured cluster network
    <img width="2195" alt="image" src="https://github.com/harvester/harvester/assets/4569037/f177d506-a2f4-43ef-b8fe-993ee95bf9d0">
1. check the MTU again, and the MTU value of `*-br` is changed to `1500`
    ```
    141: vlan-bo: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 9000 qdisc noqueue master vlan-br state UP group default qlen 1000
    142: vlan-br: <BROADCAST,MULTICAST,PROMISC,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN group default
    ```

**Expected behavior**
the MTU value should be persisted even after configured as storage network.

**Support bundle**

**Environment**
 - Harvester ISO version: v1.1.1
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): All

**Additional context**
This probably is overwritten by the [NAD bridge config](https://github.com/harvester/harvester/blob/master/pkg/controller/master/storagenetwork/storage_network.go#L267), need to check if it can support [MTU config ](https://github.com/k8snetworkplumbingwg/k8s-net-attach-def-controller/blob/master/README.md)

Temp Workaround:
run `$ip link set mtu 9000 dev $interface` to update the MTU config in all nodes.

**Comment by w13915984028** [comment]
connect with https://github.com/harvester/harvester/issues/4353

**Comment by bk201** [comment]
Supposed to be fixed in https://github.com/harvester/harvester/issues/4353, see [comment](https://github.com/harvester/network-controller-harvester/pull/87#pullrequestreview-1562143227)

**Comment by WebberHuang1118** [comment]
Verify fixed for Harvester v1.2.0 (harvester-network-controller v0.3.4)
